discovery-provider/src/queries/get_health.py

```diff
@@ -126,6 +128,12 @@ def get_health(args, use_redis_cache=True):
         latest_block_num = latest_block.number
         latest_block_hash = latest_block.hash.hex()
 
+        if use_redis_cache:
```

Comment:
there is already an `if use_redis_cache:` above on line 115, why not put these at the bottom of that block?